tap_db2/__init__.py

```diff
@@ -706,16 +770,30 @@ def do_sync(mssql_conn, config, catalog, state):
 
 def log_server_params(mssql_conn):
```

Comment:
Just a hygiene thing, it would be nice to remove all mssql definitions e.g. change mssql_conn references to db2_conn.

Reply:
Done